src/contracts/core/DelegationManager.sol

```diff
@@ -554,30 +559,31 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
                     shares: withdrawal.shares[i],
                     token: tokens[i]
                 });
-                unchecked { ++i; }
+                unchecked {
+                    ++i;
+                }
             }
-        // Award shares back in StrategyManager/EigenPodManager. If withdrawer is delegated, increase the shares delegated to the operator
+            // Award shares back in StrategyManager/EigenPodManager. If withdrawer is delegated, increase the shares delegated to the operator
```

Comment:
Don't like this but maybe the comment just needs to be in the else branch entirely